README.md

```diff
@@ -7,21 +8,31 @@ This is a PHP implementation of the GraphQL [specification](https://github.com/f
 based on the [reference implementation in JavaScript](https://github.com/graphql/graphql-js).
 
 ## Installation
+
 Via composer:
-```
-composer require webonyx/graphql-php
-```
+
+    composer require webonyx/graphql-php
```

Comment:
This used to be better

Reply:
[`4bc7beb` (#855)](https://github.com/webonyx/graphql-php/pull/855/commits/4bc7bebe0aba3d653deb69881814047116748c37)